anvill/python/anvill/binja/callingconvention.py

```diff
@@ -13,36 +13,41 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import binaryninja as bn
 
-class CallingConvention:
+from anvill.util import *
+
+_FLOAT_ARGS_REGS = {
```

Comment:
Should some of these be uppercase?

Reply:
The register's name gets changed to uppercase at a later point before the register family lookup or generating the spec. The binary ninja calling convention also gives the register's name in lower case and it gets handled accordingly at a later point.